inc/features/blocks/personalization/components/views.js

```diff
@@ -25,10 +32,18 @@ const Views = ( { isLoading, label = null, total } ) => {
 	}
 
 	return (
-		<p className="altis-analytics-views">
-			<Icon icon="visibility" />
-			{ label || sprintf( _n( '%d view', '%d views', total, 'altis-experiments' ), total ) }
-		</p>
+		<div className="altis-analytics-views">
+			<div className="altis-analytics-views__total">
+				<Icon icon="visibility" />
+				{ label || sprintf( _n( '%d view', '%d views', total, 'altis-experiments' ), total ) }
+			</div>
+			<div className="altis-analytics-views__conversions">
+				<Icon icon="yes" />
+				{ conversionsLabel || sprintf( _n( '%d conversion', '%d conversions', conversions, 'altis-experiments' ), conversions ) }
+				{ ' ' }
+				({ ( ( conversions / ( conversionsDenominator || total ) ) * 100 ).toFixed( 1 ) }%)
```

Comment:
This is a bit hairy, can we move to a var earlier to separate the calculation from the formatting?

Reply:
Yeah I actually hate this, just showing conversions for total impressions was very very awkward